src/data/faq.json

```diff
@@ -149,8 +149,8 @@
                         "active": false,
                         "textblock": [
                             "Starting with <a href='../../en/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>version 2.5</a>, the Corona-Warn-App (CWA) supports all three types of EU Digital COVID Certificate:",
-                            "<ul>  <li>digital vaccination certificate</li>  <li>digital recovery certificate</li>  <li>digital test certificate</li> </ul>",
-                            "Previously, support for digital vaccination certificates was introduced in <a href='../../en/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>version 2.3</a>, followed by the addition of support for digital test certificates in <a href='../../en/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>version 2.4.</a> The release of <a href='../../en/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>version 2.5</a> added support for digital recovery certificates.",
+                            "<ul>  <li>digital vaccination certificate</li>  <li>digital certificate of recovery</li>  <li>digital test certificate</li> </ul>",
+                            "Previously, support for digital vaccination certificates was introduced in <a href='../../en/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>version 2.3</a>, followed by the addition of support for digital test certificates in <a href='../../en/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>version 2.4.</a> The release of <a href='../../en/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>version 2.5</a> added support for digital certificates.",

```

Comment:
added support for digital certificates.",

should be

added support for digital certificates of recovery.",

Reply:
Thanks Mike! I'll fix this with the approval of our plan for the next steps from the Open Source Team. 
Enjoy the evening!

Reply:
Fixed in [`1ab092c`](https://github.com/corona-warn-app/cwa-website/pull/2263/commits/1ab092c90bbb0b31d29ee45b8476ba2e220f8ccb).